extensions/amp-a4a/0.1/template-renderer.js

```diff
@@ -64,16 +64,19 @@ export class TemplateRenderer extends Renderer {
       }
       const templateHelper = getAmpAdTemplateHelper(context.win);
       return templateHelper
-        .render(data, iframe.contentWindow.document.body)
+        .render(
+          data,
+          (iframe.contentDocument || iframe.contentWindow.document).body
```

Comment:
Maybe extracting this to a function? We would save some bytes and this seems to be a common usage.

Reply:
Would you like the function to be in a different file than the change here?